toolbox/load_aware.py

```diff
@@ -10,14 +10,41 @@ class LoadAware:
 
     @AnsibleRole("load_aware_deploy_trimaran")
     @AnsibleMappedParams
-    def deploy_trimaran(self):
+    def deploy_trimaran(self, log_level=1,
+                        plugin="TargetLoadPacking", default_requests_cpu="2000m",
+                        default_target_requests_multiplier="2", target_utilization=70,
```

Comment:
> default_target_requests_multiplier="2"
why is this one a `str`, not an `int`? 🤔 
and can you put all the parameters in a dedicated line? (just for consistency), and with a comma after the last one

Reply:
Now that is a good question! That was actually intentional. For some reason that field is a string in the config

https://github.com/kubernetes-sigs/scheduler-plugins/tree/master/pkg/trimaran/targetloadpacking

Not sure why though, will have to make sure their example is correct.

Reply:
ack, if it's on purpose, keep it as you think is most suitable!